src/styles/fonts.module.scss

```diff
@@ -0,0 +1,303 @@
+@mixin font-body-3 {
+  font-family: Karla;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
```

Comment:
Hmm feels like the font-weight should not be included in these mixins.

Reply:
Why not? 🤔